processing/src/main/java/org/apache/druid/math/expr/ExpressionProcessingConfig.java

```diff
@@ -39,9 +34,6 @@ public class ExpressionProcessingConfig
       "druid.expressions.homogenizeNullMultiValueStringArrays";
   public static final String ALLOW_VECTORIZE_FALLBACK = "druid.expressions.allowVectorizeFallback";
 
-  @JsonProperty("useStrictBooleans")
-  private final boolean useStrictBooleans;
```

Comment:
IMO, we should keep the config here, and use it for exactly one thing: to log a warning on startup if it's set to `false`. Just a little hint for people that upgrade without reading the release notes, and end up wondering why behavior changed.

Reply:
ah, that's reasonable, will add it back